CMakeLists.txt

```
@@ -291,6 +291,7 @@ if(NOT USE_XNNPACK AND CMAKE_VERSION VERSION_LESS ${XNNPACK_MIN_CMAKE_VER})
   message(WARNING "USE_XNNPACK is set to OFF. XNNPACK requires CMake version ${XNNPACK_MIN_CMAKE_VER} or greater.")
 endif()
 option(USE_ZMQ "Use ZMQ" OFF)
+option(USE_ITT "Use Intel(R) VTune Profiler ITT functionality" ON)
```

This should only be enabled for X86 CPU, isn't it?
```suggestion
cmake_dependent_option(USE_ITT "Use Intel(R) VTune Profiler ITT functionality" ON "CPU_INTEL" OFF)
```

---

this will also work for Intel GPU (XPU)

---

But it would not work on ARM64, would it?

---

Correct. Not expect it working on ARM64. I'll bound it to IntelCPU